### ISSUE TITLE
typography footnotes

### ISSUE THREAD

**Issue by br092616** [pwrstudio/rr_wp#42]
for the footnotes could you change line height to 20px, and decrease the indenting of the text by 5px also?

**Comment by pwrstudio** [comment]
line height changed

**Comment by br092616** [comment]
can we have that format also apart from footnotes, for bullet lists (with bullet points instead of the footnote numbers? put content under Understanding: Reading is a creative act to try

**Comment by br092616** [comment]
thank god there's version tracking!

**Comment by br092616** [comment]
forget about the bulletet list from wordpress, let's better do it manually. so you can take that out from the text editor, also the numbered list

**Comment by br092616** [comment]
...however for the footnotes, I think it's better if the indent remains as it is here http://52.91.41.41/guiding-question/#how-can-the-masters-tools-dismantle-the-masters-house-about-the-painting 
although the numbers don't quite line up yet, and the indent to the text could be a little less...

**Comment by manegame** [comment]
I tried the following: all lists are styled with a uniform indent (numbered and bulleted). No need to insert bullets manually, this is using the bulleted list **built into Wordpress**.

Changed the distance and alignment for lists to be a little less.

See screenshots, **NB:** this is not yet propagated on 52.92.41.41

Unfortunately hanging punctuation is not supported, so quotes will not hang.

Does this solve the issue?

<img width="718" alt="schermafbeelding 2016-09-12 om 16 12 03" src="https://cloud.githubusercontent.com/assets/17074938/18438997/e2ecb8e0-7903-11e6-9671-976596ee389e.png">
<img width="714" alt="schermafbeelding 2016-09-12 om 16 12 46" src="https://cloud.githubusercontent.com/assets/17074938/18438998/e2ed8022-7903-11e6-9e3b-eeac68910c46.png">
